src/runtime/hexagon/hexagon_device_sim.cc

```diff
@@ -0,0 +1,366 @@
+/*!
+ * Copyright 2018-2019 Qualcomm Innovation Center, Inc.
```

Comment:
We are in the process of migrating to ASF, and as a future convention we do not keep copyright messages because the code will be licensed to ASF, please check if it is OK.

Reply:
I will bring it to the attention of the right people.

Reply:
This may take a bit of time to figure out.  If we keep the copyright notices, will it block this PR?

Reply:
Likely it will be a blocker, but we can first keep reviewing and make sure things are in shape. The code can exist in your repo for now

Reply:
Will it be acceptable if we remove this single line (that says "Copyright..."), and leave everything else unchanged?  We need to have a better idea of what's acceptable before we discuss it with the legal teams.

Reply:
Yes, that is exactly what I asked for, see https://www.apache.org/legal/src-headers.html

Reply:
Thanks for the clarification.

Reply:
License header should be 
```
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

  http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
```

Reply:
What does that mean exactly?

There is this, plus an extra paragraph in our license.  You said you wanted us to remove the copyright line, and that was it.  What is the intent of this comment in the light of what had been written before?

Reply:
Correction: the first paragraph is different.  Is this what you are referring to?